### DISCUSSION TITLE
New 6 second turns poll

### DISCUSSION THREAD

**Discussion by olanti-p** [cataclysmbn/Cataclysm-BN#2263] [Polls]
Sequel to https://github.com/cataclysmbnteam/Cataclysm-BN/discussions/2114

The results of the 1st poll are quite clear, so let's pose a different question.

With some non-trivial amount of effort, it should be possible to make turn length configurable and have the game balance work for configurable turn length (e.g. have hunger/thirst/fatigue rates automatically adjust to turn length, same for crop growth and various timed character effects).

That'd likely introduce various subtle bugs that would need squashing and require expanding existing infrastructure, and all future changes would have to account for configurable turn length, but it still should be doable.

**Comment by Krwak** [comment]
Now that i read about the pros of the 6 second turns, i'm totally convinced that it would be great.
This is not sarcasm btw, i didn't play Cata before 0.E and it totally makes sense now that the game was designed for 6 second turns.

**Comment by meladath** [comment]
Having it as a configurable option would be nice, but would result in quite a different experience for the player if they mess around with it without knowing what it is.

**Comment by ghost** [comment]
I played a lot of 0.D Cataclysm but never understood the difference between 1 second and 6 second turns besides "Now I have more time to do things". It would be nice to have configurable turn length.
